drivers/hud_apr/app/models/hud_apr/generators/shared/fy2020/question_eighteen.rb

```diff
@@ -29,8 +29,13 @@ class QuestionEighteen < Base
 
           answer = @report.answer(question: table_name, cell: cell)
           adults = universe.members.where(adult_clause)
-          adults = adults.where(stayers_clause) if suffix == :annual_assessment
-          adults = adults.where(leavers_clause) if suffix == :exit
+
+          case suffix
+          when :annual_assessment
+            adults = adults.where(stayers_clause.and(a_t[:annual_assessment_expected].eq(true)))
+          when :exit
+            adults = adults.where(leavers_clause) if suffix == :exit
```

Comment:
the `if suffix == :exit` seems redundant with the case statement.  Am I missing something?

Reply:
Yep!